### ISSUE TITLE
Build failure: python311Packages.sphinxcontrib_newsfeed

### ISSUE THREAD

**Issue by toastal** [NixOS/nixpkgs#263504]
### Steps To Reproduce

1. build python311Packages.sphinxcontrib_newsfeed

### Build log

https://paste.sr.ht/~toastal/1a0d87838090073fa4d068f51a34cf9dda776cb4

### Additional context

This was part of building `khal`

### Notify maintainers

none listed

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix shell nixpkgs#nix-info --command nix-info -m
 - system: `"x86_64-linux"`
 - host os: `Linux 6.5.7, NixOS, 23.11 (Tapir), 23.11.20231019.7c9cc5a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(toastal): `""`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by antonmosich** [comment]
I discovered this yesterday and bisected the build failure to be introduced in 2b1ad99374ed5af40dd3603b63b12ad11eb19af3. Didn't get to look much further into it

**Comment by arjan-s** [comment]
Yup, got the same problem here

**Comment by antonmosich** [comment]
@natsukium do you know whether #259350 will fix this issue?

**Comment by natsukium** [comment]
That PR will not fix this problem.
We need to relax the constraint of docutils version for checking.

**Comment by arjan-s** [comment]
Sorry to ask, but is there any progress?

**Comment by antonmosich** [comment]
If I'm not mistaken 91bf7eff618ee531a5510115ebaab4890780a8c1 does fix this build failure, and will be merged with #263535, the current staging cycle.

**Comment by antonmosich** [comment]
As the mentioned PR has been merged now, `sphinxcontrib_newsfeed` (now also renamed to `sphnixcontrib-newsfeed`) builds again.
